### PR TITLE
feat: implementation of outbound sync

### PR DESCRIPTION
**Pull request by albertyfwu**
this is a quickly thrown-together implementation of outbound sync. i can do another pass after this to clean up some error handling, refactoring of code, etc.

### CODE REVIEW

File: apps/api/temporal/activities/do_sync.ts [hunk 1]
```diff
@@ -219,3 +245,74 @@ async function writeRecordsToWebhook(
...
+    syncRunId: string;
+  }
+): Promise<void> {
+  // Apply mapping
+  const customerRecords = mapInternalToCustomerRecords(fieldMapping, records);
+  if (!customerRecords.length) {
+    throw new Error('No records to write');
+  }
+
+  // TODO: Validate / throw error?
```
Comment (tomkit) [inline apps/api/temporal/activities/do_sync.ts:313]: thoughts on field mapping being part of the activities sdk? cc @lucasmarshall

**Review by tomkit — APPROVED** [review]

File: apps/api/temporal/activities/do_sync.ts (hunk 1 above)
Comment (albertyfwu) [reply apps/api/temporal/activities/do_sync.ts:313]: I think before we do that we should think about whether individual parts of the sync should live in separate activities or just be in one big activity. Currently the whole sync execution is in one activity. Future discussion?

File: apps/sample-app/supaglue-config/outbound/contact.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    //   label: 'last name',
+    // },
+    // {
+    //   name: 'title',
+    //   label: 'title',
+    // },
+  ],
+});
+
+const contactMapping = sdk.defaultFieldMapping([
```
[the remark this answers is not in the document]
Comment (albertyfwu) [reply apps/sample-app/supaglue-config/outbound/contact.ts:29]: We could use the same one here.

[the remark this answers is not in the document]
Comment (albertyfwu) [reply apps/sample-app/supaglue-config/outbound/contact.ts:29]: I'll address in follow-up PR.

**Pull request merged**